### PR TITLE
update pr-date.csv with some new detected tests and PRs

### PR DESCRIPTION
**Pull request by Kerr0220**
Resubmit to rerun the check process.

### CODE REVIEW

File: pr-data.csv [hunk 1]
```diff
@@ -992,6 +992,19 @@ https://github.com/apache/cxf,649520354501295859b692ddf1b42adb919affbb,tools/wad
 https://github.com/apache/cxf,649520354501295859b692ddf1b42adb919affbb,rt/transports/http-netty/netty-server,org.apache.cxf.transport.http.netty.server.integration.NettyServerTest.testGetWsdl,ID,,,
 https://github.com/apache/cxf,649520354501295859b692ddf1b42adb919affbb,rt/transports/jms,org.apache.cxf.transport.jms.RequestResponseTest.testRequestTopicResponseStaticQueue,ID,,,
 https://github.com/apache/dolphinscheduler,26a84b30d822b59925a82fa6724df24fd5c13213,dolphinscheduler-spi,org.apache.dolphinscheduler.spi.params.PluginParamsTransferTest.testGetParamsJson,ID,Opened,https://github.com/apache/dolphinscheduler/pull/6773,
+https://github.com/apache/druid,50963edcae70150f13520b619f167512d951a71b,core,org.apache.druid.java.util.emitter.core.EmitterTest.testBatchSplitting,ID,,,
```
Comment (xieyt2000) [inline pr-data.csv:995]: For these detected tests, do you have any logs of NonDex run as proof?

File: pr-data.csv [hunk 2]
```diff
@@ -1319,7 +1332,7 @@ https://github.com/apache/hadoop,a585a73c3e02ac62350c136643a5e7f6095a3dbb,hadoop
...
 https://github.com/apache/hadoop,a585a73c3e02ac62350c136643a5e7f6095a3dbb,hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core,org.apache.hadoop.mapred.TestLineRecordReader.testUncompressedInputDefaultDelimiterPosValue,OD,,,
 https://github.com/apache/hadoop,a585a73c3e02ac62350c136643a5e7f6095a3dbb,hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core,org.apache.hadoop.mapred.TestLineRecordReader.testUncompressedInputWithLargeSplitSize,OD,,,
-https://github.com/apache/hadoop,cc2babc1f75c93bf89a8f10da525f944c15d02ea,hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-jobclient,org.apache.hadoop.mapred.TestLocalMRNotification.testMR,ID,,,
+https://github.com/apache/hadoop,cc2babc1f75c93bf89a8f10da525f944c15d02ea,hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-jobclient,org.apache.hadoop.mapred.TestLocalMRNotification.testMR,ID,DeveloperFixed,,619e490333fa89601fd476dedac6d16610e9a52a
```
Comment (xieyt2000) [inline pr-data.csv:1335]: How did you find the test was fixed in this commit? I checked the [commit (https://github.com/apache/hadoop/commit/619e490333fa89601fd476dedac6d16610e9a52a). It only changed the markdown

Comment (Kerr0220) [reply pr-data.csv:1335]: Oh no, I simply run the script in `auto-check-fix-commit`.

File: pr-data.csv (hunk 1 above)
Comment (Kerr0220) [reply pr-data.csv:995]: Yes, they are in my vm under the `druid` folder's `.nondex`. Should I provide them in some way?

File: pr-data.csv (hunk 2 above)
Comment (xieyt2000) [reply pr-data.csv:1335]: > Oh no, I simply run the script in `auto-check-fix-commit`.

Please double check, it's unlikely that this commit fixes this test

Comment (Kerr0220) [reply pr-data.csv:1335]: Ok, thanks!

File: pr-data.csv (hunk 1 above)
Comment (xieyt2000) [reply pr-data.csv:995]: > Yes, they are in my vm under the `druid` folder's `.nondex`. Should I provide them in some way?

We want the command line output for running NonDex to show that these tests are flaky. The `.nondex` folder is hard to read.

Comment (Kerr0220) [reply pr-data.csv:995]: ok, I'm going to work on that!

Comment (Kerr0220) [reply pr-data.csv]: @xieyt2000 Hi Yuntong, I've temporarily removed other changes. Could you please check these changes to see whether they can be accepted? Appreciate that. :)

File: pr-data.csv [hunk 3]
```diff
@@ -1289,8 +1289,7 @@ https://github.com/apache/hadoop,cc2babc1f75c93bf89a8f10da525f944c15d02ea,hadoop
...
 https://github.com/apache/hadoop,14cd969b6ea1898e9db6eeb9ea5292ec4558a706,hadoop-hdfs-project/hadoop-hdfs-client,org.apache.hadoop.hdfs.web.oauth2.TestClientCredentialTimeBasedTokenRefresher.refreshUrlIsCorrect,ID,,,
 https://github.com/apache/hadoop,14cd969b6ea1898e9db6eeb9ea5292ec4558a706,hadoop-hdfs-project/hadoop-hdfs-client,org.apache.hadoop.hdfs.web.oauth2.TestRefreshTokenTimeBasedTokenRefresher.refreshUrlIsCorrect,ID,,,
-https://github.com/apache/hadoop,a585a73c3e02ac62350c136643a5e7f6095a3dbb,hadoop-hdfs-project/hadoop-hdfs-client,org.apache.hadoop.hdfs.web.TestURLConnectionFactory.testSSLFactoryCleanup,OD,,,
-https://github.com/apache/hadoop,14cd969b6ea1898e9db6eeb9ea5292ec4558a706,hadoop-common-project/hadoop-common,org.apache.hadoop.ipc.TestDecayRpcScheduler.testPriority,ID,,,
+https://github.com/apache/hadoop,14cd969b6ea1898e9db6eeb9ea5292ec4558a706,hadoop-common-project/hadoop-common,org.apache.hadoop.ipc.TestDecayRpcScheduler.testPriority,ID,Opened,https://github.com/apache/hadoop/pull/5085,
```
Comment (xieyt2000) [inline pr-data.csv:1292]: Seems that you accidentally removed one line here

**Review by xieyt2000 — APPROVED** [review]

File: pr-data.csv (hunk 3 above)
Comment (xieyt2000) [inline pr-data.csv:1292]: Seems that you accidentally removed one line here

Comment (Kerr0220) [reply pr-data.csv:1292]: 🥲oh sry

**Review by xieyt2000 — APPROVED** [review]
LTGM now. BTW, you should follow up on some of the opened PRs, if you don't know how to answer the questions, you can ask on Campuswire

**Comment by Kerr0220** [comment]
@xieyt2000 Ok 👌 Thank you for the reminding!

**Pull request merged**